pykoi/retrieval/vectordb/epsilla.py

```diff
@@ -0,0 +1,128 @@
+import os
+import numpy as np
+import types
+
+from typing import List, Union
+from langchain.embeddings.base import Embeddings
+from langchain.schema import BaseRetriever, Document
+from sklearn.decomposition import PCA
+from pyepsilla import vectordb
+
+from pykoi.retrieval.llm.constants import LlmName
+from pykoi.retrieval.vectordb.abs_vectordb import AbsVectorDb
+
+model_map = {
+    LlmName.OPENAI: 1536
+}
+
+class EpsillaRetriever(BaseRetriever):
```

Comment:
Please follow https://github.com/CambioML/pykoi/blob/main/pykoi/db/qa_database.py to add missing docstring and type hint, so when we run sphnix next time, it can auto generate docs for Epsilla.

Reply:
Added. Let me know if not clear enough.